.travis/run.sh

```diff
@@ -24,10 +24,14 @@ else
         eval "$(pyenv init -)"
     fi
     if [[ "${OPENSSL}" == "0.9.8" ]]; then
-      export PATH="$HOME/ossl-098l/bin:$PATH"
-      export CFLAGS="-I$HOME/ossl-098l/include"
-      export LDFLAGS="-L$HOME/ossl-098l/lib"
-      export LD_LIBRARY_PATH="$HOME/ossl-098l/lib"
+        OPENSSL_DIR="ossl-098l"
+    fi
+
+    if [ ! -z "$OPENSSL_DIR" ]; then
```

Comment:
I didn't want to duplicate the variable exports if I didn't have to, so this conditional is so we can put an `elif [[ "${OPENSSL}" == "1.0.0" ]]; then` in the first if branch above and then share the logic.